### PR TITLE
Add ability to query selected statistics to all relevant objects/entities

### PR DESCRIPTION
**Pull request by ronhakim**

### CODE REVIEW

**Comment by ronhakim** [comment]
I noticed that similar functions to retrieve statistics are available in `PowerFlexUtility`.
However, I think the newly implemented functions cover all types of objects/entities, are more generic and are better located, in each object's class.

**Comment by kuttattz** [comment]
@ronhakim Can you please add unit tests for these changes. You can refer https://github.com/dell/python-powerflex/blob/main/tests for the same.

**Comment by ronhakim** [comment]
@kuttattz , UT added. Please review again.

**Review by kuttattz — APPROVED** [review]

File: tests/test_acceleration_pool.py [hunk 1]
```diff
@@ -71,3 +76,16 @@ def test_acceleration_pool_delete_bad_status(self):
             self.assertRaises(exceptions.PowerFlexFailDeleting,
                               self.client.acceleration_pool.delete,
                               self.fake_ap_id)
+
+    def test_acceleration_pool_query_selected_statistics(self):
```
Comment (sachin-apa) [inline tests/test_acceleration_pool.py:80]: @ronhakim Can you please add the assertion for the unit test.

Please update the same for the other objects as well.

Comment (ronhakim) [reply tests/test_acceleration_pool.py:80]: @sachin-apa, asserts added as suggested. Please review again.

**Review by sachin-apa — APPROVED** [review]
Verified all except for replication_consistency_group , replication_pair, and sdc .. Since its a common code should be good.

LGTM.

**Review by trisha-dell — APPROVED** [review]

**Pull request merged**